processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModel.java

```diff
@@ -202,6 +202,13 @@
    */
   private boolean isPartitionLoad;
 
+  /**
+   * It directly writes data directly to nosort processor bypassing convert steps.
+   * This data comes from SDK where the input is from AVRO and AVRO Writer already
+   * converts the AVRO data to the Primitive datatypes object.
+   */
+  private boolean isAvroSDKLoad;
```

Comment:
I don't think this is required, use partition flow

Reply:
Done.